site/content/docs/commands/task-run.en.md

```diff
@@ -20,44 +20,59 @@ Generally, the steps involved in task run are:
-    --follow                         Optional. Specifies if the logs should be streamed.
-    --generate-cmd string            Optional. Generate a command with a pre-filled value for each flag.
-                                     To use it for an ECS service, specify --generate-cmd <cluster name>/<service name>.
-                                     Alternatively, if the service or job is created with Copilot, specify --generate-cmd <application>/<environment>/<service or job name>.
-                                     Cannot be specified with any other flags.
--h, --help                           help for run
-    --image string                   The location of an existing Docker image.
-                                     Mutually exclusive with -d,  --dockerfile.
-    --memory int                     Optional. The amount of memory to reserve in MiB for each task. (default 512)
-    --platform-arch string           Optional. Architecture of the task. Must be specified along with 'platform-os'.
-    --platform-os string             Optional. Operating system of the task. Must be specified along with 'platform-arch'.
-    --resource-tags stringToString   Optional. Labels with a key and value separated by commas.
-                                     Allows you to categorize resources. (default [])
-    --secrets stringToString         Optional. Secrets to inject into the container. Specified by key=value separated by commas. (default [])
-    --security-groups strings        Optional. The security group IDs for the task to use. Can be specified multiple times.
-                                     Cannot be specified with 'app' or 'env'.
-    --subnets strings                Optional. The subnet IDs for the task to use. Can be specified multiple times.
-                                     Cannot be specified with 'app', 'env' or 'default'.
-    --tag string                     Optional. The container image tag in addition to "latest".
--n, --task-group-name string         Optional. The group name of the task. Tasks with the same group name share the same set of resources.
-    --task-role string               Optional. The role for the task to use.
-```
-## Example
+Name Flags
+  -n, --task-group-name string   Optional. The group name of the task. 
+                                 Tasks with the same group name share the same set of resources. 
+                                 (default directory name)
+
+Build Flags
+      --build-context string   Path to the Docker build context.
+                               Mutually exclusive with -i, --image.
+      --dockerfile string      Path to the Dockerfile.
+                               Mutually exclusive with -i, --image. (default "Dockerfile")
+  -i, --image string           The location of an existing Docker image.
+                               Mutually exclusive with -d, --dockerfile.
```

Comment:
```suggestion
                               Mutually exclusive with -d, --dockerfile, --build-context.
```

Reply:
actually, maybe we should match the wording in the placement section. Something like `Cannot be specified with 'image' or 'dockerfile'.`

Reply:
Done! Thanks for pointing out the inconsistency, updated all the remaining flags to "Cannot be specified with --%s"